_posts/en/newsletters/2025-10-24-newsletter.md

```diff
@@ -0,0 +1,66 @@
+---
+title: 'Bitcoin Optech Newsletter #377'
+permalink: /en/newsletters/2025/10/24/
+name: 2025-10-24-newsletter
+slug: 2025-10-24-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+FIXME:bitschmidty
+
+## News
+
+- **Delving thread Determining BlockTemplate Fee Increase Using Fee Rate Diagram:** A suggested title for this
+  news item could be "Detecting block template feerate increases using cluster
+  mempool" https://delvingbitcoin.org/t/determining-blocktemplate-fee-increase-using-fee-rate-diagram/2052
+
+- **Channel Jamming Mitigation Simulation Results and Updates:** Carla Kirk-Cohen [posted][carla post]
+  to Delving Bitcoin about their simulation results and updates with the reputation algorithm
+  for their [channel jamming mitigation proposal][channel jamming bolt]. Some of the notable changes are
+  that reputation is tracked for outgoing channels, and resources are limited on incoming channels.
+
+  Using their simulator they have run both the [resource][resource attacks] and [sink][sink attacks] attacks.
+  They found that with the new updates protection against resource attacks remain intact and with sink attacks
+  attacking nodes will be quickly cut off when they misbehave. It is noted that if an attacker builds a
+  reputation and then targets a chain of nodes, only the last node is compensated. But there is a
+  significant cost to an attacker to target multiple nodes.
+
+  They think that the [channel jamming][topic channel jamming attacks] mitigation has reached a point
```

Comment:
maybe
```suggestion
  The writeup concludes that the [channel jamming][topic channel jamming attacks] mitigation has reached a point
```

Reply:
fixed in [6018a4e](https://github.com/bitcoinops/bitcoinops.github.io/pull/2531/commits/6018a4e76ed70f7d97bc505cf946714d00d8e9d8)